### PR TITLE
Uyuni can also manage SLE clients (but Salt only)

### PR DESCRIPTION
**Pull request by keichwa**
# Description

Short summary of why you created this PR

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [ ] Manager-4.1
- [x] Manager-4.2
- [x] Uyuni

# Links

Fixes https://github.com/SUSE/spacewalk/issues/12795

### CODE REVIEW

**Comment by keichwa** [comment]
I'm going to add PDFs :)

**Comment by juliogonzalez** [comment]
Seems tests are failing so... I will review when all is green and ready to be reviewed (ping me on rocket.chat :-))

**Comment by Loquacity** [comment]
> Seems tests are failing so... I will review when all is green and ready to be reviewed (ping me on rocket.chat :-))

You might be waiting a while :wink:  It's OK if the Vale test fails for the moment, we're still working on it. Just make sure the build test is green.

File: modules/client-configuration/pages/client-upgrades-major.adoc [hunk 1]
```diff
@@ -135,3 +130,16 @@ When the client next boots, it will use grub to boot the new kernel with its ini
 PXE booting is not used during this process.
 
 Approximately three minutes after the job was fetched, the client will go down for reboot.
+
+
+ifeval::[{suma-content} == true]
+[NOTE]
+====
+For clients that were registered using Salt, use the ``spacewalk/minion_script`` snippet to register the client again after migration has completed.
```
Comment (Loquacity) [inline modules/client-configuration/pages/client-upgrades-major.adoc:138]:
```suggestion
For Salt clients, use the ``spacewalk/minion_script`` snippet to register the client again after migration has completed.
```

**Review by Loquacity — APPROVED** [review]

File: modules/client-configuration/pages/client-upgrades-major.adoc [hunk 2]
```diff
@@ -135,3 +130,16 @@ When the client next boots, it will use grub to boot the new kernel with its ini
...
+
+
+ifeval::[{suma-content} == true]
+[NOTE]
+====
+For Salt clients, use the ``spacewalk/minion_script`` snippet to register the client again after migration has completed.
+====
+endif::[]
+ifeval::[{uyuni-content} == true]
+// Uyuni is Salt clients only.  Uyuni does not manage traditional clients.
```
Comment (paususe) [inline modules/client-configuration/pages/client-upgrades-major.adoc:142]: ????

Uyuni contains the traditional stack

https://build.opensuse.org/project/show/systemsmanagement:Uyuni:Stable:SLE12-Uyuni-Client-Tools

https://build.opensuse.org/project/show/systemsmanagement:Uyuni:Stable:SLE15-Uyuni-Client-Tools

**Review by Loquacity — APPROVED** [review]

**Comment by keichwa** [comment]
Pau Garcia Quiles <notifications@github.com> writes:

> ????
>
> Uyuni contains the traditional stack

Thanks for checking.  Misunderstanding by my side.  it is also properly
listed here:

https://www.uyuni-project.org/uyuni-docs/uyuni/installation/client-requirements.html

I'll adjust the PR.

-- 
Karl Eichwalder

**Comment by juliogonzalez** [comment]
I didn't review before merging so... I hope it's fine :-)

**Pull request merged**